pymc3/distributions/continuous.py

```diff
@@ -3393,45 +3339,18 @@ class Triangular(BoundedContinuous):
         Upper limit.
     """
 
-    def __init__(self, lower=0, upper=1, c=0.5, *args, **kwargs):
-        self.median = self.mean = self.c = c = at.as_tensor_variable(floatX(c))
-        self.lower = lower = at.as_tensor_variable(floatX(lower))
-        self.upper = upper = at.as_tensor_variable(floatX(upper))
-
-        super().__init__(lower=lower, upper=upper, *args, **kwargs)
+    rv_op = triangular
+    rv_op.bound_args = [0, 2]  # TODO: Find less hacking solution?
```

Comment:
@brandonwillard Is there a more elegant solution? The problem was that `BoundedContinuous` is looking for the `lower/upper` inputs based on order, and in this case it's the first and last inputs that it should use.

Reply:
Yeah, we don't want to add properties to objects like this.  As a matter of fact, `Op`s are likely to use `__slots__` in the near future, making this an error.